tests/test-http-client-transport/src/test/java/org/eclipse/jetty/http/client/RoundRobinConnectionPoolTest.java

```diff
@@ -258,6 +258,6 @@ protected void service(String target, Request jettyRequest, HttpServletRequest r
         // Opening p5 and p6 was delayed, so the opening of p7 was triggered
         // to replace p4 while p5 and p6 were busy sending their requests.
         if (transport != Transport.UNIX_DOMAIN)
-            assertThat(remotePorts.toString(), count / maxUsage, lessThanOrEqualTo(results.size()));
+            assertThat(remotePorts.toString(), results.size(), lessThanOrEqualTo(count / maxUsage));
```

Comment:
Not sure about this change.
In the example in the comment above, count=12, maxUsage=2, so you would expect only 12/2=6 connections being opened. However, results=7 connections may be opened, so the old code seems correct to me.

Reply:
You're right: the parameters were in the wrong order, but the condition was reversed too so the assertion was actually correct. At least in my logic, so I'm going to leave the parameters like they were.

But I'm going to set `result.toString()` as the message instead of `remotePorts.toString()` as that saves some manual work interpreting the results and some confusion.